packages/ord-connect/src/components/SelectWalletModal/index.tsx

```diff
@@ -92,23 +92,20 @@ export function SelectWalletModal({
         throw new Error("Magic Eden via Ordit returned no addresses.");
       }
 
-      // Magic Eden provides segwit address for sending and receiving payments
-      const segwit = magicEden.find(
-        (walletAddress) => walletAddress.format === "segwit",
-      );
-      // p2sh is used to support imported xverse wallets
-      const p2sh = magicEden.find(
-        (walletAddress) => walletAddress.format === "p2sh-p2wpkh",
+      // Magic Eden provides a segwit address by default for sending and receiving payments
+      // Imported xverse wallets will return a p2sh address for payments
+      // Payment address defaults to segwit because it is the Magic Eden Wallet default option
+      // TODO: Support importing Unisat wallet
```

Comment:
this TODO comment and "Payment address defaults to segwit because it is the Magic Eden Wallet default option" can be removed as it is self-explanatory

Imported xverse wallets will return a p2sh address for payments instead

```suggestion
      // Magic Eden provides a segwit address by default for sending and receiving payments
      // Imported xverse wallets will return a p2sh address for payments by default instead
```